source/js/foundation/inject-react/multipage-nav.js

```diff
@@ -12,7 +12,15 @@ export default (apps) => {
   );
 
   if (targetNode) {
-    let links = Array.from(document.querySelectorAll(`#multipage-nav a`)).map(
+    const multipageLinks = document.querySelectorAll(`#multipage-nav a`)
```

Comment:
Good call on checking link length! 

Since we already have an `if` statement on line 14, we can combine your code with the existing line 10-14 to something like

```jsx
const targetNode = document.querySelector(
    `#multipage-nav-mobile .container .row .col-12`
  );
  const multipageLinks = document.querySelectorAll(`#multipage-nav a`);

  if (targetNode && multipageLinks.length > 0) {
```

And leave the rest of code unchanged.

Reply:
Ahh nice, good feedback. Will implement and test shortly. Thanks Mavis!